packages/sqflite/tizen/src/database_manager.cc

```diff
@@ -0,0 +1,290 @@
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 9: {
+        auto vec = std::get<std::vector<double>>(param);
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 10: {
+        auto val = std::get<flutter::EncodableList>(param);
+        std::vector<uint8_t> vec;
+        // Only  a list of uint8_t for flutter EncodableValue is supported
+        // to store it as a BLOB, otherwise a DatabaseError is triggered
+        try {
+          for (auto item : val) {
+            vec.push_back(std::get<int>(item));
+          }
+        } catch (const std::bad_variant_access) {
+          throw sqflite_errors::DatabaseError(
+              sqflite_errors::kUnknownErrorCode,
+              "statement parameter is not supported");
+        }
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      default:
+        throw sqflite_errors::DatabaseError(
+            sqflite_errors::kUnknownErrorCode,
+            "statement parameter is not supported");
+    }
+    if (err) {
+      ThrowCurrentDatabaseError();
+    }
+  }
+}
+
+sqlite3_stmt *DatabaseManager::PrepareStmt(std::string sql) {
+  auto cache_entry = stmt_chache_.find(sql);
+  if (cache_entry != stmt_chache_.end()) {
+    sqlite3_stmt *stmt = cache_entry->second;
+    sqlite3_reset(stmt);
+    sqlite3_clear_bindings(stmt);
+    return stmt;
+  } else {
+    sqlite3_stmt *stmt;
+    int result_code =
+        sqlite3_prepare_v2(database_, sql.c_str(), -1, &stmt, nullptr);
+    if (result_code) {
+      ThrowCurrentDatabaseError();
+    }
+    if (stmt != nullptr) {
+      stmt_chache_[sql] = stmt;
+    }
+    return stmt;
+  }
+}
+
+void DatabaseManager::ExecuteStmt(DatabaseManager::Statement stmt) {
+  int result_code = SQLITE_OK;
+  do {
+    result_code = sqlite3_step(stmt);
+  } while (result_code == SQLITE_ROW);
+  if (result_code != SQLITE_DONE) {
+    ThrowCurrentDatabaseError();
+  }
+}
+
+int DatabaseManager::GetStmtColumnsCount(DatabaseManager::Statement stmt) {
+  return sqlite3_column_count(stmt);
+}
+
+int DatabaseManager::GetColumnType(DatabaseManager::Statement stmt, int iCol) {
+  return sqlite3_column_type(stmt, iCol);
+}
+
+const char *DatabaseManager::GetColumnName(DatabaseManager::Statement stmt,
+                                           int iCol) {
+  return sqlite3_column_name(stmt, iCol);
+}
+
+std::pair<Columns, Resultset> DatabaseManager::QueryStmt(
+    DatabaseManager::Statement stmt) {
+  Columns cols;
+  Resultset rs;
+  const int cols_count = GetStmtColumnsCount(stmt);
+  int result_code = SQLITE_OK;
+  for (int i = 0; i < cols_count; i++) {
+    auto cName = GetColumnName(stmt, i);
+    cols.push_back(std::string(cName));
```

Comment:
`cName` → `column_name` or just
```suggestion
    cols.push_back(std::string(GetColumnName(stmt, i)));
```